components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/cache/caches.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * API Cache
+ */
+const Cache = Java.type("org.eclipse.dirigible.components.api.cache.CacheFacade");
```

Comment:
Either all methods in the `CacheFacade` should be `static` (in order to be used in the JS/TS like this `Cache.contains()`), or this should be changed to:

```ts
const CacheFacade = Java.type("org.eclipse.dirigible.components.api.cache.CacheFacade");
const cache = new CacheFacade();

export class Cache {

    public static contains(key: string): boolean {
        return cache.contains(key);
    }
...
```